test/e2e/storage/external/external.go

```diff
@@ -247,18 +247,6 @@ func (d *driverDefinition) SkipUnsupportedTest(pattern testpatterns.TestPattern)
 		e2eskipper.Skipf("Driver %q does not support volume type %q - skipping", d.DriverInfo.Name, pattern.VolType)
 	}
 
-	supported = false
```

Comment:
should we also remove the snapshot capability from the default testpattern?

Reply:
Yes for sure. I will do that in a separate PR.